Bottom: dcb1bed3b69c333e28f3c419b2c9fa0078b1c532
Top:    3d5e9ad354c82e6a5dd0f73ba73625695e5e6363
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 02:14:41 +0100

Refresh of use-heap-structure-to-find-hol

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5a0eabd..7c21e80 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1174,7 +1174,7 @@ static void
 InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
 {
   POLYAREA *curc;
-  PLINE *curh, *container, *tmp;
+  PLINE *curh, *container;
   heap_t *heap;
   rtree_t *tree;
   int i;
